content/updates/2020-07/1-rules-UI.md

```diff
@@ -0,0 +1,28 @@
+---
+pim_announcement_img: ./img/Rules_UI.png
+pim_announcement_alt_img: "The UI to manage the rules"
+pim_announcement_audience:
+- EE
+---
+
+# New! Create and update rules from the PIM!
+::: meta-data type="New" features="Rules engine, Productivity" available="mid-July" link-to-doc="../articles/manage-your-rules.html#the-rule-builder-tab.html"
+:::
+
+To ease the use of the rules engine and boost your productivity, we created a brand new user interface to directly manage your rules in the PIM.  
+
+![New screen to manage your rules](../img/Rules_UI.png)
+
+Our rules engine enables you to automate enrichment actions on specific product selections. This new screen allows you to create, update and delete rules in a few clicks, it will definitely ease and faster the rules management.
```

Comment:
```suggestion
Our rules engine enables you to automate enrichment actions on specific product selections. This new screen allows you to create, update and delete rules in a few clicks, it will definitely ease and faster the rules management.  
```